Mastodon.API.Tests/Resources/next_and_prev_link

```diff
@@ -0,0 +1 @@
+﻿<https://friends.nico/api/v1/accounts/29/statuses?max_id=1860868>; rel="next", <https://friends.nico/api/v1/accounts/29/statuses?since_id=1874433>; rel="prev"
```

Comment:
Since this appears to be for unit testing, and all we're doing is verifying some parsing logic the links you are using could just be fake ones.